install.sh

```diff
@@ -104,17 +69,44 @@ else
   echo_info "Unable to detect system information, installation will continue."
 fi
 
+# Check if the user is on ChromeOS v117+ and is not in the VT-2 console (frecon)
+if (( "${CHROMEOS_RELEASE_CHROME_MILESTONE}" > "116" )); then
+  if ! tty | grep -q '/dev/pts/1'; then
+    echo_error "Please run the installer in the VT-2 shell."
+    exit 1;
+  fi
+fi
+
+# Check if the script is being run as root.
+if [ "${EUID}" == "0" ]; then
+  echo_error "Chromebrew should not be installed or run as root."
+  if tty | grep -q '/dev/pts/1'; then
+    echo_info "Please login as 'chronos' and restart the install."
+    exit 1;
+  fi
+  exit 1;
+fi
+
+# Warn users of the AMD segfault issue and allow them to work around it
+if grep "AuthenticAMD" /proc/cpuinfo; then
+  echo_info "If the install fails, try running 'CREW_AMD_SEGFAULT=1 exec bash --init-file <(curl -Ls git.io/vddgY)'"
```

Comment:
I think a better (less techie) name could be `CREW_AMD_INSTALL`.  Mentioning `SEGFAULT` sounds a bit too hackish.